elastic/kibana
# [Console] Fix issue with bulk requests

Fixes https://github.com/elastic/kibana/issues/146366


### Summary

The issue was that the parser was not able to detect the end of a bulk request correctly when multiple requests were sent. This PR fixes that issue.

### How to test

1. Go to Console
2. Send a bulk request with multiple requests. For example the following requests:

```
PUT /metrics-hello-world/_bulk?refresh
{"create":{}}
{"@timestamp":"2022-06-21T15:49:00Z","namespace":"namespace26"}
{"create":{}}
{"@timestamp":"2022-06-21T15:45:50Z","namespace":"namespace26"}
{"create":{}}
{"@timestamp":"2022-06-21T15:44:50Z","namespace":"namespace26"}


GET _alias

PUT <foo>/_bar/_baz%{test}
```

3. Send the request by pressing `CMD ⌘ + A` and `ENTER`
4. The response should be displayed correctly
